doc/source/myrocks/variables.rst

```diff
@@ -1964,6 +2020,9 @@ that should use read-free replication on the slave
 (that is, replication without row lookups).
 Empty by default.
 
+This variable is disabled in |Percona Server| 5.7.30-33. We recommend
```

Comment:
It seems that this variable was disabled from the beginning. Please ask George about it.

Reply:
checking with George.

Reply:
I believe it was in use, perhaps not correctly, but code paths that use it exist, so let's leave this as-is.